packages/tempus/src/Tempus_IntegratorObserverNoOp_decl.hpp

```diff
@@ -0,0 +1,63 @@
+// @HEADER
+// ****************************************************************************
+//                Tempus: Copyright (2017) Sandia Corporation
+//
+// Distributed under BSD 3-clause license (See accompanying file Copyright.txt)
+// ****************************************************************************
+// @HEADER
+
+#ifndef Tempus_IntegratorObserverNoOp_decl_hpp
+#define Tempus_IntegratorObserverNoOp_decl_hpp
+
+#include "Tempus_IntegratorObserver.hpp"
+#include "Tempus_Integrator.hpp"
+#include "Teuchos_Time.hpp"
+
+namespace Tempus {
+
+/** \brief IntegratorObserverNoOp class for time integrators.
+ *  This basic class has simple no-op functions, as all basic
+ *  functionality should be handled through other methods.
+ */
+template<class Scalar>
+class IntegratorObserverNoOp
+  : virtual public Tempus::IntegratorObserver<Scalar>
```

Comment:
I think I understand what this is doing.

But shouldn't `IntegratorObserverBasic` really be the noop observer? 

And move the output to a `Tempus_IntegratorObseverOutput` or something?

Reply:
I am fine with that.  The only downside is it is an incompatible backward change, which could require us to make changes to downstream apps.  :(